cvr/pom.xml

```diff
@@ -65,9 +65,9 @@
     used in the pipelines importer always matches the version used by the Genome
     Nexus annotation pipeline. -->
     <dependency>
-      <groupId>com.github.genome-nexus</groupId>
+      <groupId>com.github.averyniceday</groupId>
```

Comment:
reminder to update the `artifactId` before merge.

Reply:
done